Tella/Components/TextfieldView.swift

```diff
@@ -134,18 +143,17 @@ struct TextfieldView : View {
     var dividerView : some View {
         Divider()
             .frame(height: 1)
-            .background(shouldShowError ? Color(UIColor(hexValue: 0xFF2D2D)) : Color.white)
+            .background(shouldShowError || shouldShowErrorTextOnChange  ? Color(UIColor(hexValue: 0xFF2D2D)) : Color.white)
     }
     
     @ViewBuilder
     var errorMessageView : some View {
-        if let errorMessage = errorMessage, shouldShowError == true {
+        if let errorMessage = errorMessage, shouldShowError == true || shouldShowErrorTextOnChange {
```

Comment:
can we make it simpler by changing it like this: 
`     if let errorMessage, shouldShowError || shouldShowErrorTextOnChange {
`